root/etc/e-smith/templates/etc/dnsmasq.conf/80xmpp_srv_record

```diff
@@ -2,5 +2,5 @@
 # 80xmpp_srv_records
 #
 srv-host=_xmpp-client._tcp.{$DomainName},{$SystemName . '.' . $DomainName},5222,0,5
-srv-host=_xmpp-client._tcp.{$DomainName},{$SystemName . '.' . $DomainName},5223,0,5
+srv-host=_xmpps-client._tcp.{$DomainName},{$SystemName . '.' . $DomainName},5223,0,5
```

Comment:
Do you have any reference to `xmpps-client` service name definition? It seems only `xmpp-client` is [standardized](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=5222)

I propose to remove that line.

```suggestion
```

Reply:
-> xmpps-client for 5223: https://xmpp.org/extensions/xep-0368.html